### DISCUSSION TITLE
Chia 1.1.7 Does not sync

### DISCUSSION THREAD

**Discussion by matiasmone** [Chia-Network/chia-blockchain#7164] [Support]
Hello everyone, I know this topic is not new ... but I can't find the solution.
I have Win 10 chia 1.1.7 connected correctly and it does not synchronize.
Herealized a DMZ to local IP, eliminated antivirus and firewall, changed the keys, formatted with other windows and there is no case.
I have changed the DB but nothing ...
Any idea why?

Thanks

**Comment by max918** [comment]
Step1: check port
https://portchecker.co/
make sure 8444 is open for you public ip

Step2: manual add other peers
Introducer
North Asia introducer-apne.chia.net:8444
South Asia introducer-apse.chia.net:8444
Western North America: introducer-or.chia.net:8444
Eastern North America introducer-va.chia.net:8444
Europe: introducer-eu.chia.net:8444

Nodes
North America: node-or.chia.net:8444,
Europe: node-eu.chia.net:8444,
Asia: node-apne.chia.net:8444

**Reply by matiasmone** [reply]
Thanks for answering!
I have already tested the port and it is OK, also add the pairs as you recommended, but nothing ... does not synchronize ... it stays frozen in yellow and then red.
Any suggestion?

**Reply by max918** [reply]
maybe you can share your public ip here. let me and others can connect to your node.

**Reply by matiasmone** [reply]
I just reinstalled from 0 Chia, with clean nodes and data ... to no avail.

HELP!!!!

**Comment by loppefaaret** [comment]
please dont run DMZ and without a firewall! and furthermore listing your IP public! you are asking for troubles...
perhaps you could describe a bit more in depth how it is not syncing ? is it stuck on a specific block, or just slow ?

**Reply by matiasmone** [reply]
Thanks for answering!
If I regretted posting the IP ... but I can't find the problem.
It stays synchronizing at 0 / 543.175 ... if I change the database it is the same ... it connects but does not synchronize

**Reply by loppefaaret** [reply]
is this machine plotting or doing other CPU heavy stuff ?

**Reply by matiasmone** [reply]
I plot them on another machine, this one just for farming
The plotting machine does not connect to the network, I use madmax

**Reply by loppefaaret** [reply]
in you `config.yaml`file, under you users folder `\.chia\mainnet\config\` there is a "full_node:" section, and near the bottom of that section, do you have an entry called
```
  weight_proof_timeout: 360
```
if so, what is the value set to ?

**Reply by matiasmone** [reply]
Thanks ... the value is 360, the same.

I have done several tests and managed to synchronize by taking the pc to another site.
Obviously there is something between the pc and my router, at home the ports are open but that pc does not connect!
The pc that I use with MadMax, I gave her access to the internet, I installed chia and she if I synchronize without problems (linux) takes but synchronizes. Such is my anger that I went back to look for the Windows PC to connect it at my house again, but she does not want to synchronize ...

I really don't want to bother with this, I'm temporarily going to farm this way ... giving my neighbor a few beers.

**Comment by BrandtH22** [comment]
Since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
